src/classes/Feature.cls

```diff
@@ -40,5 +40,32 @@ public without sharing class Feature {
 		private set;
 	}
 
+	/*
+	* Returns a pod instance for your org
+	* Example: returns na2 if your instance points to https://na2.salesforce.com
+	*/
+	public static String instancePodName {
```

Comment:
Great addition. Few quick questions before I merge;

* What happens if people have a custom domain i.e. beaufort12.my.salesforce.com? Would you want this to return the underlying pod name?
* Any reason you haven't used the InstanceName field on the Organization object?
* Any reason you haven't followed the static caching pattern used in the other properties?